desktop/src/reports/report-filters/report-filters.component.html

```
@@ -0,0 +1,54 @@
+    }
+
+    <app-select class="report-filter__op" label="Operation" [addEmptyOption]="true"
+      [options]="operationOptions(def.type)" [optionsDisplayArray]="operationDisplay(def.type)"
+      [inputFormControl]="filterGroup | formGet: def.field + '.operation'"></app-select>
+
+    <app-button matButtonType="iconButton" icon="close" color="warn" tooltip="Remove filter"
+      (clicked)="removeFilter(def.field)" data-testid="report-filter-remove"></app-button>
+  </div>
+}
+
+@if (addableFields().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)